modules/flann/include/opencv2/flann/miniflann.hpp

```diff
@@ -143,6 +143,7 @@ struct CV_EXPORTS SavedIndexParams : public IndexParams
 
 struct CV_EXPORTS SearchParams : public IndexParams
 {
+    SearchParams( int checks, float eps, bool sorted, bool explore_all_trees = false );
     SearchParams( int checks = 32, float eps = 0, bool sorted = true );
```

Comment:
Call with 3 args is ambiguous now.
Remove default value from:

> explore_all_trees = false

Reply:
@alalek My bad. Done.
BTW, not directly related but I didn't get answer if the WIDE universal intrinsics are fully functional, or if I should stick to the "normal" universal intrinsics.